site/content/en/docs/integration/toloka.md

```diff
@@ -0,0 +1,213 @@
+CVAT, you'll need to configure a Toloka pool,
+specifying all task requirements and setting the price for task completion.
+
+To set up the Toloka pool, do the following:
+
+1. Open Toloka task, go to **Actions** > **Setup Toloka pool**.
+
+   ![Set up Toloka pool](/images/toloka08.jpg)
+
+2. In the **Create crowdsourcing task** form, fill in the following fields:
+
+   ![Set up Toloka pool](/images/toloka09.jpg)
+
+   - **Price per job**: Specify the payment amount for completing
+     one ощи within the project.
+   - **Use project description**: Switch this toggle if you want
+     to use the overarching project description for individual tasks.
+   - **Description**: Provide details about the project.
+     This field is visible only when the **Use project description** toggle is off.
+   - **Sensible content**: Switch this toggle if your dataset
+     contains images intended for an adult audience.
+   - **Accept solutions automatically**: Enable this if you
+     wish for completed jobs to be automatically accepted.
+   - **Close pool after completion, sec**: Enter the time limit, in seconds,
+     within which each job should be completed. You can also click on the
+     field to select from one of the predefined periods.
+   - **Time per task suite, sec**: Enter the time allowed for completing a task suite,
+     in seconds. Uncompleted tasks are redistributed to other Tolokers.
+   - **Days for manual review by requester**: Specify the Review period in days —
+     the number of days for the review (from 1 to 21 days from the task completion date).
+     The Toloker will see the deadline in the task information on the main Toloka page.
+   - **Audience**: Add rules to make jobs available only to Tolokers who meet certain criteria.
+     For example, you might require Tolokers to be proficient in English and have higher education.
+     These rules operate based on filter principles. For more information,
+     see [Filters](http://localhost:1313/docs/manual/advanced/filter/) documentation
+     or [CVAT Filters](https://www.youtube.com/watch?v=lj6KLIFn24A) video tutorial.
+   - ![Toloka Rules](/images/toloka10.jpg)
+
+3. Click **Submit**. You will see the **Toloka task was created** pop-up and the **Update pool** form.
+
+   ![Update Project](/images/toloka11.jpg)
+
+   - ![Open pool in Toloka](/images/toloka04.png) opens pool in Toloka.
+   - ![Open task in CVAT](/images/toloka05.png) opens task in CVAT.
+
+4. Open Toloka task, go to **Actions** > **Start Toloka pool**.
+   <br>Project, that you created will show up in Toloka.</br>
+
+   ![Toloka Project](/images/toloka12.jpg)
+
+## Changing Toloka pool
+
+To change started Toloka pool, you need to stop it first.
+
+1. Open Toloka task, **Actions** > **Stop Toloka pool**.
+2. Implement changes.
+3. Open Toloka task, go to **Actions** > **Start Toloka pool**.
+
+## Reviewing annotated jobs
+
+In case the jobs you've created are not in the **Accept solutions automatically**
```

Comment:
**Accept solutions automatically** is not parameter of CVAT jobs, it is a parameter of Toloka pool